app/src/main/java/fr/groggy/racecontrol/tv/RaceControlTvApplication.kt

```diff
@@ -9,7 +11,13 @@ class RaceControlTvApplication: Application() {
 
     override fun onCreate() {
         super.onCreate()
-
+        context = this
         AndroidThreeTen.init(this)
     }
+
+    companion object {
+        @SuppressLint("StaticFieldLeak")
+        var context: Context? = null
+            private set
+    }
```

Comment:
This can also be removed

Reply:
When removing the translation on `ChannelCardPresenter.kt`, yes.